compiler/rustc_session/src/config.rs

```diff
@@ -2468,6 +2468,15 @@ pub fn build_session_options(early_dcx: &mut EarlyDiagCtxt, matches: &getopts::M
         }
     }
 
+    if !nightly_options::is_unstable_enabled(matches)
+        && cg.force_frame_pointers == FramePointer::NonLeaf
+    {
+        early_dcx.early_fatal(
+            "`-Cforce-frame-pointers=non-leaf` also requires `-Zunstable-options` \
+                and a nightly compiler",
+        )
```

Comment:
Do we really need this? What's the concern? It's already de-facto enabled via `--target=spec.json`.

Reply:
Target specs are also unstable.

Reply:
Mind, we _could_ land it as instantly stable, I agree it has strong precedent, but then it needs to wait on a full T-compiler FCP, so "this can happen **right now** without requiring everyone to agree" was why I proposed doing things "correctly".